### DISCUSSION TITLE
Login Failure: These credentials do not match our records.

### DISCUSSION THREAD

**Discussion by NBA707** [filamentphp/filament#10392] [General]
I am following the getting started https://filamentphp.com/docs/3.x/panels/installation, and I do the artisan make:filament-user  and try to login https://filament.test/admin/login but only receive back that the credentials do not match our records.   I commented out the $this->hasVerifiedEmail(); as well            

Not sure how to get the PHP script here, Github refuses to attach the PHP script, and pasting it in removed all the line feeds.

Thanks for any thoughts
![Screenshot 2023-12-19 at 8 18 15 AM](https://github.com/filamentphp/filament/assets/52007275/f592da2e-c476-4ef4-bed3-6d6d17ebe9f6)

![Screenshot 2023-12-19 at 8 13 21 AM](https://github.com/filamentphp/filament/assets/52007275/8feeb6fa-699d-4be8-80db-2c9eaaa32a9c)

**Comment by ghost** [comment]
Did you actually comment out `$this->hasVerifiedEmail();`? I'm not seeing it in the code you shared 🤔.

**Comment by leandrocfe** [comment]
Is this account created? Did you check in the database or via tinker?
```bash
php artisan tinker
```

```bash
User::first()
```

**Comment by NBA707** [comment]
Turned out it was a trate.   I commented them out and login worked.

` // use HasApiTokens, HasFactory, Notifiable;`

**Comment by guilty-p01nt3r** [comment]
In my case the user model was different (with a different guard)
I had to append the` ->authGuard('admin');`  in Filament's configuration file: `AdminPanelProvider`

```php
class AdminPanelProvider extends PanelProvider
{
    public function panel(Panel $panel): Panel
    {
        return $panel
            ->default()
            ->id('admin')
            ->path('admin')
            ->login()
            ->colors([
                'primary' => Color::Amber,
            ])
            ->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources')
            ->discoverPages(in: app_path('Filament/Pages'), for: 'App\\Filament\\Pages')
            ->pages([
                Pages\Dashboard::class,
            ])
            ->discoverWidgets(in: app_path('Filament/Widgets'), for: 'App\\Filament\\Widgets')
            ->widgets([
                Widgets\AccountWidget::class,
                Widgets\FilamentInfoWidget::class,
            ])
            ->middleware([
                EncryptCookies::class,
                AddQueuedCookiesToResponse::class,
                StartSession::class,
                AuthenticateSession::class,
                ShareErrorsFromSession::class,
                VerifyCsrfToken::class,
                SubstituteBindings::class,
                DisableBladeIconComponents::class,
                DispatchServingFilamentEvent::class,
            ])
            ->authMiddleware([
                Authenticate::class,
            ])->authGuard('admin'); // <--- This is the only addition I made
    }
}
```
